holoviews/plotting/bokeh/callbacks.py

```diff
@@ -734,13 +734,13 @@ def _process_selection_event(self):
             code="""
             export default ({panel}, event, _) => {
               if (!event.visible) {
-                panel.position.setv({x: NaN, y: NaN})
+                panel.visible = false;
               }
             }""",
         ))
         # the first element is the close button
         self._panel.elements = [self._panel.elements[0], model]
-        if self.plot.comm:
+        if self.plot.comm:  # update on notebook
```

Comment:
```suggestion
        if self.plot.comm:  # update Jupyter Notebook
```

It's a more descriptive comment, a Notebook could be many things.  Same with the other comment.